.github/workflows/tests.yml

```diff
@@ -133,6 +122,76 @@ jobs:
                                          homebrew/cask-fonts \
                                          homebrew/cask-versions
 
+  formula-audit:
+    name: formula audit
+    needs: syntax
+    if: startsWith(github.repository, 'Homebrew/')
+    runs-on: ubuntu-22.04
+    env:
+      HOMEBREW_NO_INSTALL_FROM_API: 1
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Cache Bundler RubyGems
+        uses: actions/cache@v1
+        with:
+          path: ${{ steps.set-up-homebrew.outputs.gems-path }}
+          key: ${{ runner.os }}-rubygems-${{ steps.set-up-homebrew.outputs.gems-hash }}
+          restore-keys: ${{ runner.os }}-rubygems-
+
+      - name: Install Bundler RubyGems
+        run: brew install-bundler-gems --groups=sorbet
+
+      - name: Set up the homebrew/core tap
+        run: brew tap homebrew/core
+
+      - name: Run brew readall on homebrew/core
+        run: brew readall --aliases homebrew/core
+
+      - name: Run brew audit --skip-style on homebrew/core
+        run: brew audit --skip-style --except=version --display-failures-only --tap=homebrew/core
```

Comment:
Does it make sense to use `--display-failures-only` here? We used to only use it for the general audit and not the one of `homebrew/core`. I'm going to rerun it without this option and see if it takes longer to complete.

Reply:
Seems to run at the same speed so I think it should be fine without it.

Reply:
It does end up being kind of noisy on the cask side though.

Reply:
That's an interesting point. I'll remove that commit and then this should be good.